lib/matplotlib/patches.py

```diff
@@ -1462,6 +1462,95 @@ def get_patch_transform(self):
         return self._patch_transform
 
 
+class Annulus(Patch):
+    """
+    An elliptical annulus.
+    """
+
+    def __str__(self):
+        if self.a == self.b:
+            fmt = "Annulus(xy=(%s, %s), r=%s, width=%s, angle=%s)"
+            pars = (self.center[0], self.center[1],
+                    self.a, self.width, self.angle)
+        else:
+            fmt = "Annulus(xy=(%s, %s), a=%s, b=%s, width=%s, angle=%s)"
+            pars = (self.center[0], self.center[1],
+                    self.a, self.b, self.width, self.angle)
+        return fmt % pars
+
+    @docstring.dedent_interpd
+    def __init__(self, xy, r, width, angle=0.0, **kwargs):
+        """
+        *xy*
```

Comment:
This looks cool!  Just one small nitpick...

No doubt where you copied these docstrings from was also the old style, but these new ones should be in the new style at least:  

http://matplotlib.org/devdocs/devel/documenting_mpl.html#writing-new-documentation